docs/src/modules/utils/generateMarkdown.js

```diff
@@ -273,6 +273,14 @@ function generateProps(reactAPI) {
     return textProps;
   }, text);
 
+  let refHint = 'The `ref` is forwarded to the root element.';
+  if (reactAPI.forwardsRefTo == null) {
+    refHint = 'The component cannot hold a ref.';
```

Comment:
This is the case for almost all components that do not forward refs. Even if some of them can hold a ref we want to prevent users from doing that because converting those components to function components would be breaking.